src/index.html

```diff
@@ -3,10 +3,409 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/style.scss">
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="./images/favicon.jpg"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#moyo-web-site" class="moyo-logo-link">
+        <img
+          src="images/logo.png"
+          alt="moyo-logo"
+          class="moyo-logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple"
+              class="nav__link is-active"
+            >
```

Comment:
If you have only 2 attributes and less then 80 chars then you can leave it in one line
```suggestion
            <a href="#apple" class="nav__link is-active">
```